package.json

```diff
@@ -13,9 +13,11 @@
         "verify": "yarn lint && yarn test"
     },
     "dependencies": {
-        "@globality/nodule-config": "^2.3.0",        
+        "@globality/nodule-config": "^2.3.0",
```

Comment:
Should any of these be peer dependencies?

Reply:
I don't think any of them depend on nodule-openapi so I don't think there's a need

Reply:
Added `nodule-config` as a peer-dependency